browserstack.json

```diff
@@ -4,15 +4,31 @@
 	"test_framework": "qunit",
 	"test_path": [
 		"test/index.html",
+		"test/startError.html",
 		"test/logs.html",
-		"test/amd.html"
+		"test/amd.html",
+		"test/autostart.html"
 	],
 	"browsers": [
 		"chrome_previous",
 		"chrome_current",
 		"firefox_previous",
 		"firefox_current",
+		"opera_previous",
 		"opera_current",
+		"safari_5_1",
+		{
+			"browser": "safari",
+			"browser_version": "6.0"
+		},
+		{
+			"browser": "safari",
+			"browser_version": "7.0"
+		},
+		{
+			"browser": "safari",
+			"browser_version": "7.1"
+		},
 		"safari_previous",
 		"safari_current",
```

Comment:
7.0 is the same as 6.1, 8.0 is the same as 6.2. Unfortunately, BrowserStack removed both 7.0 & 6.1, I wrote to them we need those versions (or at least one of them).